src/pages/posts/[...slug].astro

```diff
@@ -15,12 +15,6 @@ const { post } = Astro.props;
 const { Content } = await post.render();
 ---
 
-<Layout
-  frontmatter={post.data}
-  title={post.data.title}
-  description={post.data.description}
-  image={post.data.image}
-  author={post.data.author}
->
+<Layout frontmatter={post.data}>
```

Comment:
I changed this on master so that we would actually get opengraph tags for newsletter posts.

Reply:
We were previously passing in the frontmatter and all of the frontmatter data. We can just pass the frontmatter in, then split it up inside the layout.

For the newsletters, it looks like you've added passing in the fields title, description, author, and image, which don't exist on newsletters. Do those need to be added to newsletters as well?

Reply:
Adding those extra fields was required to get the open graph and other meta tags generated

Reply:
Updated to add tests for newsletter meta tags. Newsletters have proper meta tag generation now.